pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -80,31 +80,31 @@ def v_function(params):
             dev = qml.device("default.qubit", wires=2)
 
             @qml.qnode(dev)
-            def hilbert_test(v_params, v_function, v_wires, u_tape):
-                qml.HilbertSchmidt(v_params, v_function=v_function, v_wires=v_wires, u_tape=u_tape)
-                return qml.probs(u_tape.wires + v_wires)
+            def hilbert_test(v_params, v_function, v_wires, u_script):
+                qml.HilbertSchmidt(v_params, v_function=v_function, v_wires=v_wires, u_script=u_script)
+                return qml.probs(u_script.wires + v_wires)
 
-            def cost_hst(parameters, v_function, v_wires, u_tape):
-                return (1 - hilbert_test(v_params=parameters, v_function=v_function, v_wires=v_wires, u_tape=u_tape)[0])
+            def cost_hst(parameters, v_function, v_wires, u_script):
+                return (1 - hilbert_test(v_params=parameters, v_function=v_function, v_wires=v_wires, u_script=u_script)[0])
 
         Now that the cost function has been defined it can be called for specific parameters:
 
-        >>> cost_hst([0], v_function = v_function, v_wires = [1], u_tape = u_tape)
+        >>> cost_hst([0], v_function = v_function, v_wires = [1], u_script = u_script)
         1
 
     """
     num_wires = AnyWires
     grad_method = None
 
-    def __init__(self, *params, v_function, v_wires, u_tape, do_queue=True, id=None):
+    def __init__(self, *params, v_function, v_wires, u_script, do_queue=True, id=None):
```

Comment:
I wonder if we should include a dummy key argument `u_tape` in case users are passing it by `u_tape = tape`.

Reply:
And renaming a keyword argument would probably be a breaking change.

Reply:
generally responded to concerns in the other thread.